memory-viz/src/memory_model.ts

```diff
@@ -1098,8 +1197,11 @@ export class MemoryModel {
             );
         }
 
-        const frame_types = [".frame", ".blank-frame"];
-        if (frame_types.includes(object.type) || object.type === ".class") {
+        if (object.type === ".blank" || object.type === ".blank-frame") {
+            // User must define dimensions for object with type=".blank" or ".blank-frame"
```

Comment:
Let's relax this a bit. The default values of 0 for width/height are fine; I think we can just remove the comment.

When the user doesn't specify one of these attributes, displaying a warning is appropriate but we don't need to say "User must".